src/js/controller/instream-adapter.js

```diff
@@ -146,7 +168,7 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
             _loadNextItem();
         } else {
             // notify vast of breakEnd
-            this.trigger('adBreakEnd', {});
+            this.trigger('vastAdBreakEnd', {});
```

Comment:
Why not always fire `PLAYLIST_COMPLETE` (if we skipped an ad and there are no more ads to play then we're done)? You could also change the state to COMPLETE and listen for that.